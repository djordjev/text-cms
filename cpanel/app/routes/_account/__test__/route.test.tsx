import { createRemixStub } from '@remix-run/testing';
import { render } from '@testing-library/react';

import Route from '../route';

test('route', () => {
  const createComponent = () => {
    const RemixStub = createRemixStub([
      {
        path: '/',
        Component: () => <Route />
      }
    ]);

    return <RemixStub />;
  };

  const { container } = render(createComponent());

  expect(container.childNodes).toMatchSnapshot();
});
