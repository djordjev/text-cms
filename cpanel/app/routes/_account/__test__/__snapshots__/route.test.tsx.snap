// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`route 1`] = `
NodeList [
  <div
    class="u-flex u-items-center u-justify-center u-w-full"
  >
    <div
      class="u-card u-shadow-xl u-card-bordered u-bg-secondary u-text-secondary-content md:u-w-4/12"
    >
      <div
        class="u-card-body"
      />
    </div>
  </div>,
]
`;
